app/src/main/java/at/roteskreuz/stopcorona/model/managers/DatabaseCleanupManager.kt

```diff
@@ -65,6 +53,8 @@ class DatabaseCleanupManagerImpl(
             }
 
             val nowAsRollingStartIntervalNumber = ZonedDateTime.now()
+                .minusDays(NUMBER_OF_DAYS_THE_GREEN_KEYS_ARE_KEPT)
+                .startOfTheDay()
```

Comment:
UTC day

Reply:
Please also change all occurences of `startOfTheDay` to `startOfTheUtcDay` in this file.

Reply:
updated